Makefile

```diff
@@ -30,6 +30,14 @@ ci-run: ## Provisions AWS EC2 hosts for testing staging environment.
 ci-go: ## Creates, provisions, tests, and destroys AWS EC2 hosts for testing staging environment.
 	@if [[ "${CIRCLE_BRANCH}" != docs-* ]]; then molecule test -s aws; else echo Not running on docs branch...; fi
 
+.PHONY: ci-lint-image
```

Comment:
Could you please add something that looks like the following so it can be used outside of the CircleCI context as well ?
<pre>
# unique SHA1 that may be defined by CircleCI and that is unique to a test run
CIRCLE_SHA1 ?= 1
</pre>